scripts/build_release.py

```diff
@@ -25,12 +25,14 @@ def _check_output(cmd, default=None):
 
 
 def build(env):
+    from six.moves import reload_module
     check_call(['tox', '-e', env])
     out_pyi = os.path.join(os.getcwd(), os.path.join('dist', 'console'))
-    exe = os.path.join(out_pyi, 'console')
-    # https://bugs.python.org/issue18920
-    print("Running {} to determine its version.".format(str(exe)))
-    v = _check_output([str(exe), '-V'], default="unknown")
+    # tox writes _version.py, doing a delayed import and a reload after that
+    from piksi_tools import _version
+    reload_module(_version)
```

Comment:
Don't we need to assign the result of this to `_version` in order to get a reference to the new module?

Reply:
I don't think it's needed.

> If a module imports objects from another module using from … import …, calling reload() for the other module does not redefine the objects imported from it — one way around this is to re-execute the from statement, another is to use import and qualified names (module.*name*) instead.

This uses the latter approach, i.e., `v = _version.version`.

Reply:
Sounds good